restcomm/restcomm.dao/src/main/java/org/mobicents/servlet/restcomm/entities/GeolocationList.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * TeleStax, Open Source Cloud Communications
+ * Copyright 2011-2016, Telestax Inc and individual contributors
+ * by the @authors tag.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.mobicents.servlet.restcomm.entities;
+
+import java.util.List;
+
+import org.mobicents.servlet.restcomm.annotations.concurrency.NotThreadSafe;
+
+/**
+ * @author fernando.mendioroz@telestax.com (Fernando Mendioroz)
+ *
+ */
+@NotThreadSafe
+public final class GeolocationList {
```

Comment:
ok, i just try to prevent unccesary wrapping classes around collections, specially when using generic, i guess this is following established design patter in Restcomm ;)